src/components/dataset/schemaCreation/DatasetSchemaCreationView.tsx

```diff
@@ -80,58 +86,100 @@ const styles = (theme: CustomTheme) => ({
   formLabelError: {
     color: theme.palette.error.main,
   },
+  leftMargin: {
+    marginLeft: 20,
+  },
+  flexRow: {
+    display: 'flex',
+    alignItems: 'center',
+    marginTop: 10,
+  },
 });
 
 interface IProps extends WithStyles<typeof styles> {
-  userEmail: string;
+  profiles: Array<BillingProfileModel>;
+  dispatch: Dispatch<Action>;
 }
 
 interface TabConfig {
   description: string;
   content: any;
 }
 
-const DatasetSchemaCreationView = withStyles(styles)(({ classes }: IProps) => {
+const DatasetSchemaCreationView = withStyles(styles)(({ classes, dispatch, profiles }: IProps) => {
   const [currentTab, setCurrentTab] = React.useState(0);
-  const changeTab = (_event: any, newCurrentTab: any) => setCurrentTab(newCurrentTab);
+  const ref = useRef<HTMLInputElement>(null);
+  const updateTab = (tab: number) => {
+    // Reset scroll on tab change
+    ref.current?.parentElement?.scrollTo(0, 0);
+    setCurrentTab(tab);
+  };
+  const changeTab = (_event: any, newCurrentTab: number) => updateTab(newCurrentTab);
 
   const formMethods = useForm({
-    mode: 'onTouched',
+    mode: 'all',
     reValidateMode: 'onChange',
     defaultValues: {
-      name: '',
-      description: '',
-      terraProject: '',
-      enableSecureMonitoring: 'true',
+      name: null,
+      description: null,
+      terraProject: null,
+      enableSecureMonitoring: false,
       cloudPlatform: CLOUD_PLATFORMS.gcp.key,
-      region: '',
+      defaultProfileId: null,
+      region: null,
       stewards: [],
       custodians: [],
```

Comment:
Is there a reason why we don't allow snapshot creators to be set at this stage too?

When we allow the policy members to be set here, is there an opportunity to reuse code from snapshot creation? The difference in UX threw me off a bit. 
<img width="596" alt="Screen Shot 2023-01-20 at 3 15 40 PM" src="https://user-images.githubusercontent.com/79769153/213797556-10ec331a-99ae-466a-9069-1fcebc5c2c10.png">